DAO/project/DAO-contract/tests/utils/mod.rs

```diff
@@ -130,9 +130,22 @@ pub mod abi_calls {
     }
 }
 
+pub mod paths {
```

Comment:
I would define the paths at the top instead of above the section that uses it because it's easier to navigate to the top rather than somewhere in the file. You need to know where the relevant section is to look directly above it. This is fine though.